feathr_project/feathr/protobuf/featureValue_pb2.py

```diff
@@ -0,0 +1,165 @@
+from google.protobuf import descriptor_pool as _descriptor_pool
+from google.protobuf import message as _message
+from google.protobuf import reflection as _reflection
+from google.protobuf import symbol_database as _symbol_database
+# @@protoc_insertion_point(imports)
+
+_sym_db = _symbol_database.Default()
+
+
+
+
+DESCRIPTOR = _descriptor_pool.Default().AddSerializedFile(b'\n\x12\x66\x65\x61tureValue.proto\x12\x08protobuf\"\xb5\x06\n\x0c\x46\x65\x61tureValue\x12\x16\n\x0c\x62ooleanValue\x18\x01 \x01(\x08H\x00\x12\x15\n\x0bstringValue\x18\x02 \x01(\tH\x00\x12\x14\n\nfloatValue\x18\x03 \x01(\x02H\x00\x12\x15\n\x0b\x64oubleValue\x18\x04 \x01(\x01H\x00\x12\x12\n\x08intValue\x18\x05 \x01(\x05H\x00\x12\x13\n\tlongValue\x18\x06 \x01(\x03H\x00\x12.\n\x0c\x62ooleanArray\x18\x08 \x01(\x0b\x32\x16.protobuf.BooleanArrayH\x00\x12,\n\x0bstringArray\x18\t \x01(\x0b\x32\x15.protobuf.StringArrayH\x00\x12*\n\nfloatArray\x18\n \x01(\x0b\x32\x14.protobuf.FloatArrayH\x00\x12,\n\x0b\x64oubleArray\x18\x0b \x01(\x0b\x32\x15.protobuf.DoubleArrayH\x00\x12*\n\x08intArray\x18\x0c \x01(\x0b\x32\x16.protobuf.IntegerArrayH\x00\x12(\n\tlongArray\x18\r \x01(\x0b\x32\x13.protobuf.LongArrayH\x00\x12)\n\tbyteArray\x18\x0e \x01(\x0b\x32\x14.protobuf.BytesArrayH\x00\x12\x38\n\x11sparseStringArray\x18\x0f \x01(\x0b\x32\x1b.protobuf.SparseStringArrayH\x00\x12\x34\n\x0fsparseBoolArray\x18\x10 \x01(\x0b\x32\x19.protobuf.SparseBoolArrayH\x00\x12:\n\x12sparseIntegerArray\x18\x11 \x01(\x0b\x32\x1c.protobuf.SparseIntegerArrayH\x00\x12\x34\n\x0fsparseLongArray\x18\x12 \x01(\x0b\x32\x19.protobuf.SparseLongArrayH\x00\x12\x38\n\x11sparseDoubleArray\x18\x13 \x01(\x0b\x32\x1b.protobuf.SparseDoubleArrayH\x00\x12\x36\n\x10sparseFloatArray\x18\x14 \x01(\x0b\x32\x1a.protobuf.SparseFloatArrayH\x00\x42\x13\n\x11\x46\x65\x61tureValueOneOf\" \n\x0c\x42ooleanArray\x12\x10\n\x08\x62ooleans\x18\x01 \x03(\x08\"\x1e\n\x0bStringArray\x12\x0f\n\x07strings\x18\x01 \x03(\t\"\x1e\n\x0b\x44oubleArray\x12\x0f\n\x07\x64oubles\x18\x01 \x03(\x01\"\x1c\n\nFloatArray\x12\x0e\n\x06\x66loats\x18\x01 \x03(\x02\"\x1c\n\x0cIntegerArray\x12\x0c\n\x04ints\x18\x01 \x03(\x05\"\x1a\n\tLongArray\x12\r\n\x05longs\x18\x01 \x03(\x03\"!\n\nBytesArray\x12\x13\n\x0b\x62ytes_array\x18\x01 \x03(\x0c\"6\n\x11SparseStringArray\x12\x10\n\x08integers\x18\x01 \x03(\x05\x12\x0f\n\x07strings\x18\x02 \x03(\t\"5\n\x0fSparseBoolArray\x12\x10\n\x08integers\x18\x01 \x03(\x05\x12\x10\n\x08\x62ooleans\x18\x02 \x03(\x08\"D\n\x12SparseIntegerArray\x12\x16\n\x0eindex_integers\x18\x01 \x03(\x05\x12\x16\n\x0evalue_integers\x18\x02 \x03(\x05\"8\n\x0fSparseLongArray\x12\x16\n\x0eindex_integers\x18\x01 \x03(\x05\x12\r\n\x05longs\x18\x02 \x03(\x03\"6\n\x11SparseDoubleArray\x12\x10\n\x08integers\x18\x01 \x03(\x05\x12\x0f\n\x07\x64oubles\x18\x02 \x03(\x01\"4\n\x10SparseFloatArray\x12\x10\n\x08integers\x18\x01 \x03(\x05\x12\x0e\n\x06\x66loats\x18\x02 \x03(\x02\x42+\n)com.linkedin.feathr.common.types.protobufb\x06proto3')
```

Comment:
Add some comments on where this is from (or how to update it)?

Reply:
this is generated.